SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -705,41 +697,6 @@ void USpatialNetDriver::MakePlayerSpawnRequest()
 	}
 }
 
-void USpatialNetDriver::OnLevelAddedToWorld(ULevel* LoadedLevel, UWorld* OwningWorld)
```

Comment:
From what I understand, this is called when a streamed level is added to the world. Where is the actor initialization done without this callback ?

Reply:
My impression was that this what also result in calling `OnLevelInitializedNetworkActors`

Reply:
Seems right. Although the LevelAddedToWorld delegate is an original one, while the LevelInitializedNetworkActor is one we added. Makes me wonder why we need it.